ResidentsSocialCarePlatformApi.Tests/V1/Helper/TestHelper.cs

```diff
@@ -167,5 +169,15 @@ public static PersonalRelationship CreatePersonalRelationship(long? personId = n
                 .RuleFor(personalRelationship => personalRelationship.ParentalReponsibility, f => f.Random.String2(1, "YN"))
                 .RuleFor(personalRelationship => personalRelationship.IsInformalCarer, f => f.Random.String2(1, "YN"));
         }
+
+        public static ResidentsSocialCarePlatformApi.V1.Domain.PersonalRelationships CreateRandomPersonalRelationship()
```

Comment:
Would it be easier to add using for ResidentsSocialCarePlatformApi.V1.Domain? Faster to type and easier to read if using only the class name

Reply:
Added :)